core/platform/models.py

```diff
@@ -27,8 +27,17 @@
 
 
 class _Platform(object):
+    """Acts as a base class for platform-specific imports related to GAE.
+    """
     @classmethod
     def import_models(cls):
+        """Acts as an abstract method that should be implemented on inherited
+        classes.
+
+        Raises:
+            NotImplementedError: method not overwritten in derived classes.
+
+        """
```

Comment:
Remove blank line before """.

Reply:
Okay, fixed that :smile:. Doing proper documentation is hard :relaxed:

Reply:
Yes, initially it might be. But after you know everything, it's easy to keep check of.